packages/cli/medusa-cli/src/create-cli.ts

```diff
@@ -59,6 +64,15 @@ function buildLocalCommands(cli, isLocalProject) {
   }
 
   cli
+    .command({
+      command: "types",
+      handler: handlerP(
+        getCommandHandler("gql-to-ts", (args, cmd) => {
+          process.env.NODE_ENV = process.env.NODE_ENV || `development`
+          return cmd(args)
+        })
+      ),
+    })
```

Comment:
Should we remove this command for now and just rely on dev-server to generate the needed types.

Let's see if people have any custom use-cases for manual generation?

Reply:
Yes i was thinking of that too, lets do that